src/main/kotlin/com/zepben/evolve/database/sqlite/cim/tables/iec61970/base/wires/TableRotatingMachines.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.database.sqlite.cim.tables.iec61970.base.wires
+
+import com.zepben.evolve.database.sqlite.cim.tables.Column
+import com.zepben.evolve.database.sqlite.cim.tables.Column.Nullable.NULL
+
+@Suppress("PropertyName")
+abstract class TableRotatingMachines : TableRegulatingCondEq() {
+
+    val RATED_POWER_FACTOR: Column = Column(++columnIndex, "rated_power_factor", "NUMBER", NULL)
+    val RATED_S: Column = Column(++columnIndex, "rated_s", "NUMBER", NULL)
+    val RATED_U: Column = Column(++columnIndex, "rated_u", "NUMBER", NULL)
```

Comment:
todo: Should be updated to an int as per my comment on the data spec.

Reply:
changed in table, class and changeset

Reply:
you also changed `rated_s`, which should have remained as a `NUMBER`. Please revert the change to `rated_s`, but keep the change to `rated_u`. This can be confirmed if you look at the data spec.

Reply:
Changed it back, Originally did it to the data class too which picked up the error, missed that the table needed to be changed back too.